docs/.gitignore

```diff
@@ -1,3 +1,7 @@
 /.vuepress/.cache/
 /.vuepress/.temp/
 /apidocs/
+
+/CONTRIBUTING.md
+/index.md
```

Comment:
Formely, it's just a symlink. But vitepress doesn't process symlink correctly. We need to copy the file instead.